mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -519,7 +519,8 @@ struct Conv2DRewritePattern : public OpRewritePattern<T> {
     // filter : K & CRS , if CRS is under 64 or 32
     // we pad CRS to 32 or 64, then mlir can do gemm
     // we add more one transform to do pad
-    if (convOpType == miopen::ConvOpType::Conv2DOpType && gemmKExtra > 0) {
+    if (convOpType == miopen::ConvOpType::Conv2DOpType &&
```

Comment:
Per discussion, logic concerning padding along K/M/N dimensions shall Not be included in this PR